components/articles/articleAuthor.tsx

```diff
@@ -1,14 +1,14 @@
 "use client";
-
 import Image from "next/image";
 
```

Comment:
Add next/link import
```suggestion
import Link from "next/link";

```